compiler/src/iree/compiler/Codegen/Passes.h

```diff
@@ -575,6 +575,16 @@ std::unique_ptr<OperationPass<ModuleOp>> createSPIRVEmulateI64Pass();
 std::unique_ptr<OperationPass<func::FuncOp>>
 createSPIRVMapMemRefStorageClassPass();
 
+/// Pass pipeline to lower winograd ops. This pipeline follows the
+/// SPIRVBaseVectorize pipeline with the following exception:
+/// Since the ops are already tiled, we skip tiling and instead
+/// just annotate the loops with the spirv distribute attribute.
+///
+void addSPIRVWinogradVectorizePassPipeline(OpPassManager &pm);
```

Comment:
Not a blocker, but is there a way to not make this a separate pass pipeline?

Reply:
I think so but the tradeoff then is that I would have to make "enableWinograd" a part of the TranslationInfoAttr so that in the SPIRVLowerExecutablePass, in the switch case to determine the lowering pipeline, I can call addSPIRVBaseVectorizePassPipeline(pipeline, enableWinograd) where enableWinograd is queried from translationInfo. Which do you think is better - separate pass pipeline or new parameter to translation info attr?

Reply:
I think separate pipelines is good. Some pipelines (e.g., SPIRVBaseVectorize) we have right now are way overloaded for various purposes; I'm looking to clean them up to avoid all the branchy cases inside, as it makes verification easier, etc. (We can have commonly shared subsequence of passes in helper functions though.)

Reply:
Sounds good, will leave it as is then.